common/src/test/java/com/github/dominik48n/party/config/DocumentTest.java

```diff
@@ -80,5 +92,65 @@ void testReadNonExistentFile() {
         final File file = new File("non-existent.json");
         assertThrows(FileNotFoundException.class, () -> Document.read(file));
     }
+
+    @Test
+    public void testAppendAndGetList() {
+        List<String> list = of("Value1", "Value2", "Value3");
+
+        document.append("key", list, Document::addJsonConsumer);
+
+        List<String> retrievedList = document.getList("key", new ArrayList<>(), Document.stringCollector());
```

Comment:
```suggestion
        final List<String> retrievedList = document.getList("key", new ArrayList<>(), Document.stringCollector());
```
We want to be sure that the list is not overwritten.